tests/context.py

```diff
@@ -85,6 +94,24 @@
 except ImportError as err:  # pragma: no branch
 	raise ModuleNotFoundError("[CWE-440] unittest Failed to import.") from err
 
+try:
+	if 'contextlib' not in sys.modules:
+		import contextlib
+	else:  # pragma: no branch
+		contextlib = sys.modules["""contextlib"""]
+except ImportError as err:  # pragma: no branch
+	raise ModuleNotFoundError("[CWE-440] contextlib Failed to import.") from err
+
+
+try:
+	if 'contextlib.contextmanager' not in sys.modules:
+		from contextlib import contextmanager
+	else:  # pragma: no branch
+		contextlib.contextmanager = sys.modules["""contextlib.contextmanager"""]
+		contextmanager = contextlib.contextmanager
+except ImportError as err:  # pragma: no branch
+	raise ModuleNotFoundError("[CWE-440] contextlib.contextmanager Failed to import.") from err
+
```

Comment:
This will be simplified to:
```suggestion
try:
	from contextlib import contextmanager
except ImportError as err:  # pragma: no branch
	raise ModuleNotFoundError("[CWE-440] contextlib.contextmanager Failed to import.") from err
```
for now, and refactored as part of #151 with additional regression testing